src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,11 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+class LetterRemover private constructor() {
+
+    companion object {
+        @JvmStatic fun removeLetter(input: String): CharArray = input
+            .map { if (it.lowercaseChar() == 'r') '!' else it.lowercaseChar() }
```

Comment:
Please check this method
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html

Reply:
done